lib/compilers/fortran.ts

```diff
@@ -32,7 +33,7 @@ export class FortranCompiler extends BaseCompiler {
         return 'fortran';
     }
 
-    async runCompiler(compiler, options, inputFilename, execOptions) {
+    override async runCompiler(compiler, options, inputFilename, execOptions): Promise<CompilationResult> {
```

Comment:
I think we can grab parameter types from https://github.com/compiler-explorer/compiler-explorer/blob/a436592401f5155a6ae8bef5ad5ece324c8ea4ac/lib/base-compiler.ts#L346